docs/api.md

```diff
@@ -61,6 +61,8 @@ cols: the number of columns that the display has
 
 rows: the number of rows that the display has
 
+charsize: the number of dots that the display has (default: LCD_5x8DOTS)
```

Comment:
```suggestion
charsize (optional): the number of dots the display has per character: LCD_5x8DOTS for 5x8, LCD_5x10DOTS for 5x10. (default: LCD_5x8DOTS)
```
- Mark parameter as optional (following the convention from https://github.com/arduino-libraries/LiquidCrystal/blob/master/docs/api.md#parameters-6)
- Specify it is number per character (as opposed to the number for the entire display)
- List valid values (following the convention from https://github.com/arduino-libraries/LiquidCrystal/blob/master/docs/api.md#parameters-6)